.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Report an issue with the InfluxData documentation.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+_Describe the issue here._
+
+##### Relevant URLs
```

Comment:
```suggestion
##### Add relevant URLs
```

Reply:
This is a heading that renders after the issue is created. It would seem odd to have "Add relevant URLs" in all our our newly created issues.